WalletWasabi/WabiSabi/CoordinatorParameters.cs

```diff
@@ -35,6 +35,11 @@ public CoordinatorParameters(string dataDir)
 	/// </summary>
 	public string CoinJoinIdStoreFilePath => Path.Combine(CoordinatorDataDir, "CoinJoinIdStore.txt");
 
+	/// <summary>
+	/// File that is storing the feeRate statistics.
+	/// </summary>
+	public string CoinJoinFeeRateStatStoreFilePath => Path.Combine(CoordinatorDataDir, "CoinJoinFeeRateStatStore.txt");
```

Comment:
If possible, I would change it to a getter, this always allocates uselessly.

```suggestion
	public string CoinJoinFeeRateStatStoreFilePath { get; } = Path.Combine(CoordinatorDataDir, "CoinJoinFeeRateStatStore.txt");
```

Reply:
Cannot fix this, compilation error and it goes deeper. Let's consider this out of the scope of this PR.